net.labymod.LabyMod.yml

```diff
@@ -0,0 +1,36 @@
+id: net.labymod.LabyMod
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: labymodlauncher
```

Comment:
I don't see this command anywhere.

Reply:
The file should be located in build/files/bin/labymodlauncher